packages/server/src/user/__tests__/user.controller.spec.ts

```diff
@@ -0,0 +1,166 @@
+import { HttpException } from '@nestjs/common';
+import { Project, User } from '@prisma/client';
+import { PrismaService } from '../../prisma/prisma.service';
+import { UserController } from '../user.controller';
+import { UserService } from '../user.service';
+import { UserTestUtil } from './utils/user.test.util';
+
+describe('UserModule Integration Test', () => {
+  let userTestUtil: UserTestUtil;
+
+  let prisma: PrismaService;
+  let userController: UserController;
+  let userService: UserService;
+
+  let dummyProjects: Project[];
+  let dummyAdmins: User[];
+  let dummyUsers: User[];
+
+  let randomProject: Project;
+  let randomUser: User;
+
+  beforeAll(async () => {
+    userTestUtil = new UserTestUtil();
+
+    const moduleRef = await userTestUtil.setup();
+
+    prisma = moduleRef.get(PrismaService);
+
+    dummyProjects = await userTestUtil.createDummyProjects();
+    dummyAdmins = await userTestUtil.createDummyAdmins();
+    dummyUsers = await userTestUtil.createDummyUsers();
+
+    // create controller and service objects
+    userService = new UserService(prisma);
+    userController = new UserController(userService);
+  });
+
+  afterAll(async () => {
+    await userTestUtil.tearDown();
+  });
+
+  beforeEach(async () => {
+    randomProject = dummyProjects[Math.floor(Math.random() * dummyProjects.length)];
+    randomUser = dummyUsers.concat(dummyAdmins)[Math.floor(Math.random() * (dummyAdmins.length + dummyUsers.length))];
+  });
+
+  describe('Test cases for getMyInfo() API', () => {
```

Comment:
NIT: You don't need Test Case here, when fails it will concatenate all describes and testcase names

Therefore the Failing test would be named: `UserModule Integration Test Test cases for getMyInfo() API Return requested user object (self)`

We can just name these:
```suggestion
  describe('getMyInfo() API', () => {
```

Reply:
Changed